DOCSP-39286 document requirement for gcc 12+ (#17)
mongodb/docs-pymongo-arrow: source/faq.txt

This page contains frequently asked questions and their answers.

> Why do I get ``ModuleNotFoundError: No module named 'polars'`` when using {+driver-short+}?

Reviewer: I: [Titles need headline-style caps](https://www.mongodb.com/docs/meta/style-guide/style/titles-and-headings/#capitalization)

```suggestion
Why Do I Get ``ModuleNotFoundError: No module named 'polars'`` When Using {+driver-short+}?
```

.. code-block:: sh

   $ python -m pip install polars

> Why do I get an ``undefined symbol`` error when installing {+driver-short+} from source?

Reviewer: ```suggestion
Why Do I Get an ``undefined symbol`` Error When Installing {+driver-short+} from Source?
```

.. code-block:: sh

   $ python -m pip install polars

Why do I get an ``undefined symbol`` error when installing {+driver-short+} from source?

> --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------

Reviewer: This looks like too many underlines. Can you check it's the correct number?

> If you experience this error, check if you have GCC version 12 or later
> installed. {+driver-short+} raises this error when GCC version 12 or later is
> not installed in your Linux environment. To learn more about requirements to
> install {+driver-short+} from source, see :doc:`Installing and Upgrading
> <installation>`.

Reviewer: Suggestion:

```suggestion
{+driver-short+} raises this error when GCC version 12 or later is
not installed in your Linux environment. If you experience this error,
ensure that you have GCC version 12 or later
installed. 

To learn more about the requirements for
installing {+driver-short+} from source, see :doc:`Installing and Upgrading
<installation>`.
```

> 

Reviewer: Delete extra blank lines
